<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GSM Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenToAdd"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionModeBasic"/>
    <addaction name="actionModeLayer"/>
    <addaction name="actionModeSketch"/>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDeleteEdge"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionVoronoi"/>
    <addaction name="actionShowArea"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionControlWidget"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuSelect"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTool"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/New_24x24.png</normaloff>:/MainWindow/Resources/New_24x24.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Open_24x24.png</normaloff>:/MainWindow/Resources/Open_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpenToAdd">
   <property name="text">
    <string>Open To Add</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Save_24x24.png</normaloff>:/MainWindow/Resources/Save_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionModeBasic">
   <property name="text">
    <string>Basic</string>
   </property>
  </action>
  <action name="actionModeLayer">
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
  <action name="actionModeSketch">
   <property name="text">
    <string>Sketch</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Undo_24x24.png</normaloff>:/MainWindow/Resources/Undo_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Cut_24x24.png</normaloff>:/MainWindow/Resources/Cut_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Copy_24x24.png</normaloff>:/MainWindow/Resources/Copy_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Paste_24x24.png</normaloff>:/MainWindow/Resources/Paste_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDeleteEdge">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Resources/Delete_24x24.png</normaloff>:/MainWindow/Resources/Delete_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionVoronoi">
   <property name="text">
    <string>Voronoi</string>
   </property>
  </action>
  <action name="actionShowArea">
   <property name="text">
    <string>Show Area</string>
   </property>
  </action>
  <action name="actionControlWidget">
   <property name="text">
    <string>Control Widget</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
